### DISCUSSION TITLE
No longer possible to select location at sea in Stellarium Web

### DISCUSSION THREAD

**Discussion by Asgi80** [Stellarium/stellarium#2866] [Q&A]
Until last week I was able to select a location at sea in Stellarium Web. That seems not possible anymore. I can only select location on land or close to land. Anyone aware of any changes here or is this a bug?

Thanks

**Comment by gzotti** [comment]
I confirm it appears to choose the nearest known location, and fails setting some location in the middle of the Atlantic. Suspected bugs should however be reported to the https://github.com/Stellarium/stellarium-web-engine repository.

**Reply by Asgi80** [reply]
A!  I will report it as you specified. Thank you.
